### DISCUSSION TITLE
可否提供划词+快捷键翻译

### DISCUSSION THREAD

**Discussion by somnifex** [immersive-translate/immersive-translate#896] [Ideas]
### 新功能 | New Feature

鼠标选中文字，点按特定快捷键例如复用alt键，可以翻译选中区域

### 为什么您想要添加这些功能 | Why you want to add these features

很多时候并不需要完整翻译，而是针对某个句子、某个词，这时候可以翻译特定词句。如果划词就翻译的话可能干扰其他阅读(例如有人喜欢把滑动当做阅读指示器），所以需要引入某个特定的快捷键。alt在现有功能中能设置为翻译鼠标悬停部分，可以复用为划词后alt可以翻译选择部分

**Comment by KnIfER** [comment]
划词已在“抹布翻译”用户脚本中实现（欢迎讨论）：#1916
